config/default/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-        - image: gcr.io/openshift-gce-devel/cluster-api-operator-amd64:dev
+        - image: karthikkn27/capioperator:v2
```

Comment:
This shouldn't be merged while using this image

Reply:
Yeah, updated it in latest commit, By mistake i updated the file with test data values